src/pages/Quiz.tsx

```diff
@@ -6,54 +6,42 @@ import {
   Row,
   Table,
 } from 'react-bootstrap';
-import type Quiz from '../types/Quiz';
 import useModal from '../hooks/useModal';
 import { QuizForm } from '../features/quiz/ui/QuizForm';
-import useQuizStore from '../store/useQuizStore';
 import { useState } from 'react';
 import QuizSearchBar from '../features/quiz/ui/QuizSearchBar';
-import quizzesQueries from '../queries/quizzes';
+import quizzesQueries from '../features/quiz/queries';
+import type { Mod, Quiz, QuizFilters } from '../features/quiz/types';
 export default function Quiz() {
-  const [query, setQuery] = useState<{ sectionId?: number; partId?: number }>(
-    {}
-  );
-  const { data: quizzes, isLoading } = quizzesQueries.read(query);
-  const createMutation = quizzesQueries.create();
-  const updateMutation = quizzesQueries.update();
+  const [filters, setFilters] = useState<QuizFilters>({
+    partId: 0,
+    sectionId: 0,
+  });
+  const { data: quizzes, isLoading } = quizzesQueries.read(filters);
   const deleteMutation = quizzesQueries.delete();
-  const [mod, setMod] = useState<'create' | 'update'>();
   const { isShow, closeModal, openModal, Modal } = useModal();
-  const { quiz, resetQuiz, setQuiz } = useQuizStore();
+  const [quiz, setQuiz] = useState<Omit<Quiz, 'sectionId'>>();
+  const [mod, setMod] = useState<Mod>('create');
   if (isLoading) {
     return <div>로딩 중...</div>;
   }
   if (!quizzes) {
     return <div>404...</div>;
   }
+
   return (
     <>
       <Modal
         isShow={isShow}
-        closeModal={closeModal}
         title="퀴즈 생성"
-        closeEvent={resetQuiz}
-        submitEvent={() => {
-          switch (mod) {
-            case 'create':
-              createMutation.mutate(quiz as Quiz);
-              break;
-            case 'update':
-              updateMutation.mutate(quiz as Quiz);
-          }
-
-          resetQuiz();
-        }}
+        closeModal={closeModal}
+        closeEvent={() => setQuiz(undefined)}
```

Comment:
```suggestion
        closeEvent={() => setQuiz(null)}
```
undefined와 null의 차이를 혹시 아실까요?

Reply:
아이고.. 보통 의도적으로 비워놓을땐 null을 많이 사용하는데 제대로 생각하지 않고 코드를 짠 것 같습니다 😢